craft_providers/lxd/lxd_provider.py

```diff
@@ -110,13 +110,13 @@ def launched_environment(
         :param project_name: Name of project.
         :param project_path: Path to project.
         :param base_configuration: Base configuration to apply to instance.
-        :param build_base: Base to build from.
+        :param build_base: Base to build from. (Deprecated)
         :param instance_name: Name of the instance to launch.
         :param allow_unstable: If true, allow unstable images to be launched.
 
         :raises LXDError: if instance cannot be configured and launched.
         """
-        image = get_remote_image(build_base)
+        image = get_remote_image(base_configuration)
```

Comment:
What do you think about:
```
    if build_base:
        logger.warning(
            "Deprecated: Parameter 'build_base' is deprecated and should not be used. "
            "The build base now comes from the base_configuration's alias."
        )
```

Reply:
I was thinking about that, but maybe create too much noisy?